diff --git a/node_modules/@indielayer/vue-live-demo/src/VueLiveEditor.vue b/node_modules/@indielayer/vue-live-demo/src/VueLiveEditor.vue
index 4e23d08..98b45d6 100644
--- a/node_modules/@indielayer/vue-live-demo/src/VueLiveEditor.vue
+++ b/node_modules/@indielayer/vue-live-demo/src/VueLiveEditor.vue
@@ -152,5 +152,6 @@ button:active {
 
 .btn-reset {
   background-color: #4a4a45;
+  display: none;
 }
 </style>
